apps/framework-cli/src/framework/core/infrastructure_map.rs

```diff
@@ -0,0 +1,213 @@
+use hyper::http::Method;
+use std::{collections::HashMap, time::Duration};
+
+use crate::framework::data_model::{
+    config::EndpointIngestionFormat,
+    schema::{DataModel, Table},
+};
+
+use super::primitive_map::PrimitiveMap;
+
+type TopicId = String;
+type APIId = String;
```

Comment:
https://rust-lang.github.io/api-guidelines/naming.html

> In UpperCamelCase, acronyms and contractions of compound words count as one word: use Uuid rather than UUID

BTW I've never found type alias for simple types aid readability.

Reply:
Yeah I might just get rid of that next PR - with the ids that makes more sense